develop/TOOLS/DefinitionOfDone/About_DoD.md

```diff
@@ -2,11 +2,11 @@
 uid: About_DoD
 ---
 
-# Definition of Done (DoD) tool
+# Definition of Done (DoD)
```

Comment:
I think it might be better to add "platform" here as well (and for the sake of consistency possibly also in the toc.yml). This for people who come across this page from the search, so they immediately have a better idea of what the page is about despite the lack of context, as otherwise they might initially think it's about the concept "definition of done".

Reply:
done.
On the page itself I added "platform" starting lower case as it made more sense to me and tool also used to be lower case (feel free to change as you see fit)
However, on the toc, I added "Platform" starting with upper case for consistency with "Skyline Driver Passport Platform" just next to it. I would feel like using lower case but then if felt we would also have to adapt "Skyline Driver Passport platform" for consistency and wasn't sure if I should (again, feel free to adapt as you see fit)